pkg/sql/plan/base_binder.go

```diff
@@ -851,6 +851,10 @@ func bindFuncExprImplByPlanExpr(name string, args []*Expr) (*plan.Expr, error) {
 		}
 	}
 
+	if name == "uuid" {
+		args = append(args, makePlan2Int64ConstExprWithType(1))
```

Comment:
use null will be better. because this parameter will not be used

Reply:
it's ok